webapp/app/components/header/components/AppLinks.js

```diff
@@ -20,6 +20,7 @@ const AppLinks = ({ i18n }) => {
       className="app-header__menu-item">
       FRA Platform
     </NavLink>
+    <div className="nav__divider"/>
```

Comment:
This should be visible only to when the menu after is visible

Reply:
Can you explain?

Reply:
Currently looks like:
![image](https://user-images.githubusercontent.com/5508251/76070223-7327ad00-5f94-11ea-92f2-5f67545ed0ed.png)

Reply:
Is the divider visible in statisticalfactsheets view? It should not